bin/codemods/run.js

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env node
+
+/**
+ * External dependencies
+ */
+const fs = require( 'fs' );
+const path = require( 'path' );
+const child_process = require( 'child_process' );
+const glob = require( 'glob' );
+
+/**
+ * Internal dependencies
+ */
+const config = require( './src/config' );
+
+const LOCAL_CODEMOD_PATH = 'bin/codemods/src/';
+
+function getLocalCodemodFileNames() {
+	// Returns all JS files in bin/codemods/src folder, except for the config file.
+	return glob.sync( '!(config).js', { cwd: LOCAL_CODEMOD_PATH } )
```

Comment:
was this change a net win? seemed pretty clear before with the entire path in there vs. now where we have to look up and guess at what `cwd` means (is it the same as I expect?) and if that constant is also used somewhere else

Reply:
Hmm...

Reply:
Thanks for bringing this up. After giving it some thought, I think this will be much more readable:

```js
function getLocalCodemodFileNames() {
	// Returns all JS files in bin/codemods/src folder, except for config and helpers.
	return glob.sync( 'bin/codemods/src/!(config).js' )
		.map( name => path.basename( name, '.js' ) ) // strip path and extension from filename
}
```